tracker/tracker/src/main/app/observer/observer.ts

```diff
@@ -311,6 +316,10 @@ export default abstract class Observer {
     if (cmt !== undefined) {
       return cmt;
     }
+    if (node.parentNode) {
+      const parentID = this.app.nodes.getID(node.parentNode);
+      if (parentID && this.app.sanitizer.isMaskedContainer(parentID)) return false;
+    }
```

Comment:
Move this to the `_commitNode(...)` method. There are already several checks on parentID

Reply:
Also it would be logically more clear since it will reach `this.app.sanitizer.isMaskedContainer(id)` check only when the parent is not.